docs/user-documentation/file_viewers.md

```diff
@@ -37,10 +37,10 @@ Create a Repository Item, and tag it with "Open Seadragon" (under Display Hints)
 
 ### Components of this scenario: 
 
-1. A viewer (e.g. the module OpenSeadragon)
+1. A viewer that is a field formatter  (e.g. the module OpenSeadragon)
```

Comment:
I would probably say "A viewer module that includes a field formatter". 

E.g. The OpenSeadragon module includes a field formatter that uses the OpenSeadragon viewer.